### PR TITLE
💄(frontend) create form object styles and related components

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose

With Joanie, we need form to create / update some data. In order to do that, we decided to create a set of reusable form components.

Capture d’écran 2021-12-02 à 11 17 29

## Proposal

- [x] Create form styles
- [x] Create a form scheme within `settings/_colors` 
- [x] Create reusable React form components

### CODE REVIEW

File: src/frontend/js/components/Form/Inputs.tsx [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  return (
+    <p className={classList}>
+      {children}
+      {message && <span className="form-field__message">{message}</span>}
+    </p>
+  );
+};
+
+// - TextField
+
```
Comment (mbenadda) [inline src/frontend/js/components/Form/Inputs.tsx:42]: Some comments have linebreaks between them and the item they're commenting, and some do not have linebreaks.

File: src/richie/apps/core/templates/richie/icons.html [hunk 2]
```diff
@@ -83,9 +83,13 @@
       <path d="M32 7.075c-1.175 0.525-2.444 0.875-3.769 1.031 1.356-0.813 2.394-2.1 2.887-3.631-1.269 0.75-2.675 1.3-4.169 1.594-1.2-1.275-2.906-2.069-4.794-2.069-3.625 0-6.563 2.938-6.563 6.563 0 0.512 0.056 1.012 0.169 1.494-5.456-0.275-10.294-2.888-13.531-6.862-0.563 0.969-0.887 2.1-0.887 3.3 0 2.275 1.156 4.287 2.919 5.463-1.075-0.031-2.087-0.331-2.975-0.819 0 0.025 0 0.056 0 0.081 0 3.181 2.263 5.838 5.269 6.437-0.55 0.15-1.131 0.231-1.731 0.231-0.425 0-0.831-0.044-1.237-0.119 0.838 2.606 3.263 4.506 6.131 4.563-2.25 1.762-5.075 2.813-8.156 2.813-0.531 0-1.050-0.031-1.569-0.094 2.913 1.869 6.362 2.95 10.069 2.95 12.075 0 18.681-10.006 18.681-18.681 0-0.287-0.006-0.569-0.019-0.85 1.281-0.919 2.394-2.075 3.275-3.394z"></path>
     </symbol>
 
-    <svg id="icon-arrow-right" viewBox="0 0 16 16" fill="currentColor">
+    <symbol id="icon-arrow-right" viewBox="0 0 16 16" fill="currentColor">
```
Comment (mbenadda) [inline src/richie/apps/core/templates/richie/icons.html:86]: 👍

**Review by mbenadda — COMMENTED** [review]
I think the help/error messages are too small and might be inaccessible.

I'm not sure if it's a problem for RGAA compliance or not but it looks like it could be difficult to read for some users.

Other than that I just had a small nitpick with comments

**Comment by jbpenrath** [comment]
> I think the help/error messages are too small and might be inaccessible.
> 
> I'm not sure if it's a problem for RGAA compliance or not but it looks like it could be difficult to read for some users.
> 
> Other than that I just had a small nitpick with comments

@mbenadda You're right, there is some accessibility issues that I try to fix.

- Fix RGAA color contrast issue 
- Upgrade message font size to improve readability
- Add icon next to message to precise context (error or information) for color blind.

**Review by mbenadda — APPROVED** [review]
👍

**Pull request merged**